Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -11,10 +11,7 @@
 import com.earth2me.essentials.utils.VersionUtil;
 import net.ess3.api.IEssentials;
 import net.ess3.api.MaxMoneyException;
-import net.ess3.api.events.AfkStatusChangeEvent;
-import net.ess3.api.events.JailStatusChangeEvent;
-import net.ess3.api.events.MuteStatusChangeEvent;
-import net.ess3.api.events.UserBalanceUpdateEvent;
+import net.ess3.api.events.*;
```

Comment:
Don't use * imports

Reply:
That's just what IntelliJ auto optimize did ;-;

Reply:
you can turn it off